scripts/zkai_minimal_d128_block_boundary_wrapper_gate.py

```
@@ -0,0 +1,938 @@
+import argparse
+import copy
+import csv
+import hashlib
+import io
+import json
+import os
+import pathlib
+import stat
+import sys
+import tempfile
+from typing import Any, Callable
```

Ruff (0.15.13)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)